kong/db/schema/entities/ca_certificates.lua

```diff
@@ -45,11 +45,11 @@ return {
         local now = ngx.time()
 
         if not_after < now then
-          return nil, "certificate expired, \"Not After\" time is in the past"
+          return nil, "certificate " .. entity.id .. " expired, \"Not After\" time is in the past"
         end
 
         if not cert:get_basic_constraints("CA") then
-          return nil, "certificate does not appear to be a CA because " ..
+          return nil, "certificate " .. entity.id .. " does not appear to be a CA because " ..
                       "it is missing the \"CA\" basic constraint"
```

Comment:
Isn't this a deviation from what is done in other custom_entity_check functions in core entity schemas? None of them seem to include ID of the entity in the error response from the schema library.

The issue here is more widespread. Whenever validation fails, the logs never tell you which specific resource failed validation and we should consider adding a systemic fix rather than solving specifically for CA certificates.

What do you think?